src/Controller/ReviewController.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Contractor;
+use App\Entity\Reservation;
+use App\Entity\Review;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ReviewController extends AbstractController
+{
+    /**
+     * @Route("/api/reservation/{id}/review/star/{starCount}", name="review_star", methods="GET")
```

Comment:
```suggestion
     * @Route("/api/reservation/{id}/review/star/{starCount}", name="review_star", methods="GET", requirements={"starCount"="[1-5]"})
```
Try this instead of first regex. Route will be not found if more than 5 entered